rosidl_generator_py/resource/_msg.py.em

```diff
@@ -211,7 +211,7 @@ bound = 2**nbits
 @[      end if]@
 @[  elif field.type.string_upper_bound]@
             ((isinstance(value, str) or isinstance(value, UserString)) and
-             len(value) <= @(field.type.string_upper_bound))
+             len(value.encode('utf-8')) <= @(field.type.string_upper_bound))
```

Comment:
Since we currently don't specify the encoding in the design doc I don't this can assume `utf-8` here.